settings/base.py

```diff
@@ -89,9 +89,8 @@
 
 REST_FRAMEWORK = {
     "DEFAULT_AUTHENTICATION_CLASSES": (
-        "rest_framework.authentication.TokenAuthentication",
+        "rest_framework.authentication.BasicAuthentication",
```

Comment:
@autn this is interesting - why do we need `BasicAuthentication` here also?

Reply:
@YPCrumble humm I think we don't need